### PR TITLE
[CDS-86041] CDS 818xx release notes

### PR DESCRIPTION
**Pull request by deepakbolangady**
# Harness Developer Pull Request
Thanks for helping us make the Developer Hub better. The PR will be looked at
by the CODEOWNERS. 

## What Type of PR is This?

- [ ] Issue
- [ ] Feature
- [ ] Maintenance/Chore
- [x] Release notes

If tied to an Issue, list the Issue(s) here:

* *Issue(s)*

## House Keeping
Some items to keep track of. Screen shots of changes are optional but would help the maintainers review quicker. 

- [x] Tested Locally
- [ ] *Optional* Screen Shoot.

### CODE REVIEW

File: release-notes/continuous-delivery.md [hunk 1]
```diff
@@ -51,6 +51,71 @@ import Kustomizedep from '/release-notes/shared/kustomize-3-4-5-deprecation-noti
...
+- Harness did not support expressions to identify manifest Ids provided as runtime inputs. Consequently, you could not reference Helm chart metadata in your deployments. The issue occurred when you used multiple Helm chart manifests. (CDS-84663)
+
+  This issue has been fixed. You can now access Helm chart metadata before deployment by using the expression `<+manifests.MANIFEST_ID.helm.fieldName>`. In the expression, replace `MANIFEST_ID` and `fieldName` with the appropriate values. If you use multiple Helm charts, the primary manifest is used as runtime input.
+  
+  This feature is behind the feature flag `CDS_HELM_FETCH_CHART_METADATA_NG`. To enable this feature, contact [Harness Support](mailto:support@harness.io).
+  <!-- Reviewed and approved by Pratysuh Goel in Jira. -->
+
+- If shell script execution fails with an exception such as a step timeout, the delegate logs include the message “Exception in script execution”. This message does not help attempts to determine the root cause. (CDS-85024, ZD-54110)
+
+  This issue has been fixed. The delegate logs now include a more meaningful message.
```
Comment (brian-f-harness) [inline release-notes/continuous-delivery.md:83]:
```suggestion
  This issue has been fixed. The delegate logs now include a more meaningful message. 
  
  This item requires Harness Delegate version 23.11.818xx. For information about features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate).
```

File: release-notes/early-access.md [hunk 2]
```diff
@@ -161,9 +171,19 @@ The following early access (beta) features are available for the Harness Cloud C
...
+
+* **Release date:** December 2023
+* **Release version:** 818xx
+* **Issue number:** CDS-82763
+* **Feature flag:** `CDS_ECS_BG_VALIDATION`
+* **How to enable:** Contact [Harness Support](mailto:support@harness.io)
+
+If green services exist in your Blue Green deployment, you can configure Harness to update those services instead of deleting them and then re-creating them with a new manifest and artifact. Updating existing green services is beneficial because new containers come up before old ones go down. For more information, go to [Update green services](/docs/continuous-delivery/deploy-srv-diff-platforms/aws/ecs/ecs-deployment-tutorial/#update-green-services).
+
+Additionally, before beginning the deployment, Harness validates the blue and green services based on the target group and tags them appropriately. If the validation fails, Harness aborts the deployment. For more information, go to [ECS blue/green service validations](/docs/continuous-delivery/deploy-srv-diff-platforms/aws/ecs/ecs-deployment-tutorial/#ecs-blue-green-service-validation).
```
Comment (brian-f-harness) [inline release-notes/early-access.md:187]:
```suggestion
 This item requires Harness Delegate version 23.11.818xx. For information about features that require a specific delegate version, go to the [Delegate release notes](/release-notes/delegate).
```

Comment (brian-f-harness) [inline release-notes/early-access.md:187]: @deepakbolangady Do we include these items in the main release notes too under the Early access heading? I didn't see it there.

**Comment by brian-f-harness** [comment]
@deepakbolangady I noticed there are two items on the CG list but didn't see them included here, CDS-83173 and CDS-85223.

**Comment by deepakbolangady** [comment]
> @deepakbolangady I noticed there are two items on the CG list but didn't see them included here, CDS-83173 and CDS-85223.

Hey Brian. I'd missed committing them. They are here now.

File: release-notes/early-access.md (hunk 2 above)
Comment (deepakbolangady) [reply release-notes/early-access.md:187]: Are we? The new release notes guidelines on the wiki did not have a guideline specifically about whether or not we should include EA items in the module release notes. Nevertheless, I copied them over now. I might not have looked the wiki over keenly enough. Thanks for the question.

**Pull request merged**